plugins/analytics-module-ga4/src/util/DeferredCapture.ts

```diff
@@ -0,0 +1,112 @@
+import ReactGA from 'react-ga4';
+
+import { UaEventOptions } from 'react-ga4/types/ga4';
+
+type Hit = {
+  data: {
+    hitType: 'pageview' | 'event';
+    [x: string]: any;
+  };
+};
+
+const PageViewEvent = 'pageview';
```

Comment:
Are these still relevant?  My understanding was that `pageview` is now `page_view` ([e.g.](https://support.google.com/analytics/answer/9234069#page_view)), and that custom events no longer have to be grouped together under `event`, but rather, they can have arbitrary names ([e.g.](https://developers.google.com/analytics/devguides/collection/ga4/events?client_type=gtag)).